__tests__/app.test.js

```diff
@@ -147,4 +147,49 @@ describe("/api/articles/:article_id/comments", () => {
             expect(body).toBeSortedBy("created_at")
         })
     })
+})
+
+describe("/api/articles/:article_id/comments", () => {
+    test("POST: 201 - responds with newly posted comment", () => {
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send({username: "lurker", body: "This article is sick"})
+        .expect(201)
+        .then(({body}) => {
+            expect(body.comment).toMatchObject({
+                author: "lurker", 
+                body: "This article is sick", 
+                article_id: 1, 
+                votes: 0, 
+                comment_id: expect.any(Number), 
+                created_at: expect.any(String)})
+        })
```

Comment:
You could add another test for a status 201, ignores unnecessary properties, for example:
```
{
username: "lurker",
body: "This article is sick",
extraProperty: "extra property"
}
```